db/migrate/20190127000000_enable_pgcrypto.rb

```diff
@@ -0,0 +1,5 @@
+class EnablePgcrypto < ActiveRecord::Migration[5.2]
+  def change
+    enable_extension 'pgcrypto'
```

Comment:
A superuser is required to enable this. This should be done by an admin and not by the application's db user, no?

Reply:
I don't think so? We expect dbs to have super user access anyway? FWIW, this is the recommended way to do stuff like this.